### ISSUE TITLE
Bookmarks: [Win IDE] "Set Bookmark 0" shortcut does not work

### ISSUE THREAD

**Issue by SmithTom6304** [YoYoGames/GameMaker-Bugs#2226]
### Description

In IDE v2022.5.0.8  Runtime v2022.5.0.8, the Set Bookmark 0 shortcut is broken on Windows. All other shortcut setters/getters work. Set Bookmark 0 works correctly on Mac.

### Expected Change

Set Bookmark 0 should set bookmark 0 for the given window.

### Steps To Reproduce

1. Start GameMaker
2. Open a project
3. Open a script
4. Press Ctrl, Shift, 5 to set bookmark 5 for the script editor. Observe the bookmark sets correctly, with a 5 appearing in the upper right corner.
5. Press Ctrl, Shift, 0 to set bookmark 0.
6. Observe nothing happens.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2022.5.0
2023.8.2

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

No

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by SmithTom6304** [comment]
[Contact us package](https://github.com/YoYoGames/GameMaker-Bugs/files/13390216/Bookmark0.zip)

**Comment by SmithTom6304** [comment]
Moved to April due to low priority.

**Comment by SmithTom6304** [comment]
Ctrl, Shift, 0 might be a windows system hotkey.

**Comment by CalumLaverock** [comment]
Set/Get Bookmark 0 have been removed as the "Set" hotkey is being blocked by a windows system hotkey, the other bookmark hotkeys are unaffected

**Comment by mgeddesGM** [comment]
verified for win Beta IDE v2024.800.0.597  Runtime v2024.800.0.620

**Comment by mgeddesGM** [comment]
verified in  IDE v2024.800.0.600  Runtime v2024.800.0.623 for  ctrl/cmd+0 now disabled

**Comment by YYDan** [comment]
Reopening, as we're still trying to do something with this one on IDE startup, as every IDE run quietly logs to the ui.log:

```
[12:05:06:957(12a7)] Failed registering shortcut - hotkey 'Set Bookmark 0' could not be found
[12:05:06:959(12a7)] Failed registering shortcut - hotkey 'Get Bookmark 0' could not be found

```
Beta IDE v2024.800.0.609  Runtime v2024.800.0.633

_Expected:_ Nothing should be logged, as we should not be trying to set/get this bookmark location.

**Comment by YYDan** [comment]
Correction: we log it out on _alternating_ IDE starts, because for some reason every other start nowadays logs language server stuff and misses all of the relevant lines for this.

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.800.0.614  Runtime v2024.800.0.637
